main.go

```diff
@@ -14,6 +14,7 @@ func main() {
 	defer stop()
 
 	if err := cli.New().ExecuteContext(ctx); err != nil {
-		fmt.Printf("%+v\n", err)
+		fmt.Fprintf(os.Stderr, "\n%v\n", err)
+		os.Exit(1)
```

Comment:
`os.Exit` skips `defer`.

we might want to restructure call order or add wrap it into another function.
ref: https://github.com/golang/go/issues/38261

Reply:
I think it is fine for our case. The line `ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)` is just to handle interrupt with keyboard.

I have checked the behaviour and it also return status code 1.

Reply:
fixed stew